test/sdk/OwnedClosureTest.ooc

```diff
@@ -0,0 +1,57 @@
+use ooc-base
+use ooc-unit
+import math
+
+MyClass: class {
+	content: Int { get set }
+	init: func { this content = 0 }
+	apply: func (action: Func (This)) {
+		action(this)
+		(action as Closure) free(Owner Receiver)
+	}
+}
+
+OwnedClosureTest: class extends Fixture {
+	init: func {
+		super("OwnedClosure")
+		adder1 := 1
+		adder2 := 2
+		adder3 := 3
+		function1 := func (instance: MyClass) {
+			instance content += adder1
+			instance content += adder2
+			instance content += adder3
+		}
+		function2 := func (instance: MyClass) {
+			instance content += adder1
+			instance content += adder2
+			instance content += adder3
+		}
+		this add("Owner Sender", func {
+			f := function1 as Closure
+			instance1 := MyClass new()
+			expect(f owner == Owner Receiver)
+			f = f take()
+			expect(f owner == Owner Sender)
+			instance1 apply(f as Func(MyClass))
+			(f as Func(MyClass))(instance1)
+			f free(Owner Sender)
+			expect(instance1 content, is equal to(12))
+			instance1 free()
+		})
+		this add("Owner Receiver", func {
+			f := function2 as Closure
+			instance1 := MyClass new()
+			expect(f owner == Owner Receiver)
+			f = f give()
+			f free(Owner Sender)
+			expect(f owner == Owner Receiver)
+			(f as Func(MyClass))(instance1)
+			instance1 apply(f as Func(MyClass))
+			expect(instance1 content, is equal to(12))
+			instance1 free()
```

Comment:
I assume you split these up afterwards? Anyway, just call them `instance` without the `1`.